### DISCUSSION TITLE
HugeCTR Slack Channel

### DISCUSSION THREAD

**Discussion by minseokl** [NVIDIA-Merlin/HugeCTR#425] [Announcements]
We have created a Slack channel to better answer questions form our users while discussing how to enhance HugeCTR from various perspectives.Please join us via [this link](https://join.slack.com/t/hugectr/shared_invite/zt-2ji0b305s-SIVB~_XZYtz38JCkT8VFSg).

**Comment by klmentzer** [comment]
Hello - is this slack channel still active? The invite link seems to have expired. Thanks!

**Reply by minseokl** [reply]
Hi @klmentzer The link was stale. I've just updated it. Please check. Thanks!
